cea/optimization/optimization_main.py

```diff
@@ -10,10 +10,12 @@
 import cea.inputlocator
 from cea.optimization.prices import Prices as Prices
 import cea.optimization.distribution.network_opt_main as network_opt
-import cea.optimization.master.master_main as master
+import cea.optimization.master.master_main as new_master
```

Comment:
why? why `new_master`? what's "new" about it? in 6 months time, will it still be "new"? do we need to import ... "as"? only do this if relying heavily (multiple times) on the contents of the imported module. also, what is wrong with the name `master_main`? at least that is the same as the module...